experimental/PlaneCurve/PlaneCurve.jl

```diff
@@ -8,7 +8,8 @@ export Point, ideal_point, AffinePlaneCurve, ProjPlaneCurve, hash, degree,
 
 ################################################################################
 
-abstract type PlaneCurve end
+abstract type PlaneCurve{S <: Union{RingElem, FieldElem}} end
```

Comment:
```suggestion
abstract type PlaneCurve{S <: RingElem} end
```
Since `FieldElem <: RingElem`.